### PR TITLE
[10.0] [MIG] account_invoice_currency

### PR DESCRIPTION
**Pull request by seb4itik**

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
Hum ... The CLA is signed for a long time ago :smile:

**Review by leemannd — APPROVED** [review]
@seb4itik Hello, thanks for the port. Code review and functional testing ok. Just some nitpicking in PR https://github.com/seb4itik/account-financial-tools/pull/2

**Comment by leemannd** [comment]
@pedrobaeza Just a question. Do we need to keep the migration folder for the migration folder from V8 to V9 ? https://github.com/OCA/account-financial-tools/pull/443/files#diff-78eee12e504b8b0e19511433871ed957

**Comment by leemannd** [comment]
@seb4itik I was forgetting. Is it possible to squash a bit the commits (especially the ones for the translations)?

**Review by guewen — APPROVED** [review]

File: account_invoice_currency/tests/test_account_invoice_currency.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountInvoiceCurrency(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super(TestAccountInvoiceCurrency, self).setUp(*args, **kwargs)
```
Comment (leemannd) [inline account_invoice_currency/tests/test_account_invoice_currency.py]: super(TestAccountInvoiceCurrency, self).setUp()

File: account_invoice_currency/tests/test_account_invoice_currency.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountInvoiceCurrency(TransactionCase):
+    def setUp(self, *args, **kwargs):
```
Comment (leemannd) [inline account_invoice_currency/tests/test_account_invoice_currency.py]: def setUp(self):

File: account_invoice_currency/migrations/9.0.1.0.0/pre-migrate.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+column_renames = {
+    'account_invoice': [
+        ('cc_amount_tax', 'amount_tax_signed'),
+    ],
+}
+
+column_copies = {
+    'account_invoice': [
+        ('cc_amount_untaxed', 'amount_untaxed_signed', None),
+        ('cc_amount_total', 'amount_total_company_signed', None),
```
Comment (adrienpeiffer) [inline account_invoice_currency/migrations/9.0.1.0.0/pre-migrate.py]: As these fields already exist in the database. The copy of these may produce a crash.

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Review by leemannd — COMMENTED** [review]
For the tests, you can call them without "args,..." no?

File: account_invoice_currency/migrations/9.0.1.0.0/pre-migrate.py (hunk 3 above)
Comment (pedrobaeza) [reply account_invoice_currency/migrations/9.0.1.0.0/pre-migrate.py]: These scripts should be removed in this branch. They are from 9.0 branch.

**Comment by leemannd** [comment]
@seb4itik There is some little cosmetic changes as PR on your branch in order to make it fits OCA's standards https://github.com/seb4itik/account-financial-tools/pull/2

**Comment by pedrobaeza** [comment]
Please squash together all the "OCA transbot" commits.

**Comment by pedrobaeza** [comment]
@seb4itik are you going to work on this?

**Comment by pedrobaeza** [comment]
I have amended comments from @adrienpeiffer, so we can merge this old PR for having ready this little module.

**Comment by leemannd** [comment]
Closing my PR as it is outdated https://github.com/seb4itik/account-financial-tools/pull/2

**Pull request merged**